internal/distro/rhel9/images.go

```diff
@@ -336,6 +337,7 @@ func edgeRawImage(workload workload.Workload,
 		return nil, err
 	}
 	img.PartitionTable = pt
+	img.SysrootReadOnly = true
```

Comment:
This is already set on line 323...

Reply:
Thanks.  This must have been a rebase / merge-conflict leftover.